packages/web/components/ui/command-palette.tsx

```
@@ -0,0 +1,753 @@
+                    ) : (
+                      <FileTextIcon className='h-4 w-4' />
+                    )}
+                  </span>
+                  <div className='flex-1 min-w-0'>
+                    <div className='font-medium truncate'>
+                      {result.note.title}
+                    </div>
+                    {result.snippet && (
+                      <div
+                        className='text-xs text-muted-foreground mt-1 line-clamp-2'
+                        dangerouslySetInnerHTML={{ __html: result.snippet }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)